### PR TITLE
Add argument for starting node

### LINKED ISSUE #8: No way to specify starting node
`eularian.eularian_path` accepts a `start` argument and will automatically generate a random starting point if none is given. But `main.py` hardcodes `1`.

PR forthcoming.

### PR DESCRIPTION
**Pull request by balsama**
Fixes #8

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -17,14 +17,16 @@ def setup_args():
     """ Setup argparse to take graph name argument. """
     parser = argparse.ArgumentParser(description='Find an Eularian Cicruit.')
     parser.add_argument('graph', nargs='?', help='Name of graph to load')
+    parser.add_argument('start', nargs='?', help='The staring node. Random if none provided.')
```
Comment (supermitch) [inline main.py:20]: I think you can add `type=int` argument to `add_argument` and eliminate the need for `int(args.start)` later?

File: main.py [hunk 2]
```diff
@@ -46,8 +48,10 @@ def main():
     else:
         graph = original_graph
 
+    # If a starting node was given as an argument, start there. Else pass None and one will be randomly selected.
```
Comment (supermitch) [inline main.py:51]: I think the comment is redundant, new code is quite clear as-is.

**Review by supermitch — CHANGES_REQUESTED** [review]
Couple of suggestions.

File: main.py [hunk 3]
```diff
@@ -48,8 +48,7 @@ def main():
     else:
         graph = original_graph
 
-    # If a starting node was given as an argument, start there. Else pass None and one will be randomly selected.
-    start = int(args.start) if args.start else None
+    start = args.start if args.start else None
```
Comment (supermitch) [inline main.py:51]: Now I'm wondering if this is necessary at all? Is `args.start == None` if it is not set when executing? I think we can leave it too though, looks nice and explicit.

**Review by supermitch — APPROVED** [review]
L51 maybe not needed? Approved.

File: main.py (hunk 3 above)
Comment (balsama) [reply main.py:51]: Oh dang. I'm not too familiar with Python 😬. I think you're correct. I'll test and update in a bit.

**Pull request merged**